internal/provider/types_role.go

```diff
@@ -41,10 +41,18 @@ type orgRoleType struct {
 	UpdatedAt    types.String `tfsdk:"updated_at"`
 }
 
-type roleDatasourceType struct {
+type spaceRoleDatasourceType struct {
+	Type      types.String `tfsdk:"type"`
+	User      types.String `tfsdk:"user"`
+	Space     types.String `tfsdk:"space"`
```

Comment:
`space_guid` & `org_guid` would be more precise right (Here and in the data source)?

Reply:
The general convention we have followed is that if only space or org is mentioned by default it means only guid but if something like space_name or org_name is present like in app resource then it implies something other than guid.